aten/src/ATen/native/Sorting.cpp

```diff
@@ -214,7 +369,7 @@ void quantile_impl(
   at::lerp_out(result, values_below, values_above, weights);
 
   if (!ignore_nan) {
-    // If there are nan values and nans are not ignored then 
+    // If there are nan values and nans are not ignored then
```

Comment:
nit: "If NaNs are not ignored (i.e. quantile, not nanquantile is called) then if a NaN is present in a reduced row all the quantiles for that row are also NaN."